src/test/common/test_mutex.cc

```diff
@@ -24,14 +24,23 @@ namespace ceph
   }
 }
 
-void do_init() {
-  static CephContext* cct = nullptr;
+static CephContext* cct;
```

Comment:
This still needs initialization to null, as far as I can see

Reply:
Global variables are always initialized to 0, according to C11 standard. I'm fairly sure that's the case in c++ too. Handy stackoverflow article here:

https://stackoverflow.com/questions/16015656/are-global-variables-always-initialized-to-zero-in-c

Reply:
Thanks, I have learned something new!